website/content/en/configurations/deploy-in-openshift.md

```diff
@@ -0,0 +1,52 @@
+---
+type: docs
+title: "Setup Secrets Store CSI Driver on Azure RedHat OpenShift (ARO)"
+linkTitle: "Setup Secrets Store CSI Driver on Azure RedHat OpenShift (ARO)"
+weight: 6
+description: >
+  How to setup Azure Keyvault Provider for Secrets Store CSI Driver on Azure RedHat OpenShift (ARO) 
+---
+
+### Installation
+
+1. Install the Azure Keyvault provider for Secrets Store CSI Driver on Azure RedHat OpenShift run:
+
+    ```bash
+    helm repo add csi-secrets-store-provider-azure https://raw.githubusercontent.com/Azure/secrets-store-csi-driver-provider-azure/master/charts
+    helm install csi csi-secrets-store-provider-azure/csi-secrets-store-provider-azure --set linux.privileged=true
+    ```
+
+    Setting `linux.privileged=true` with `helm install` will enable privileged mode for the Linux *daemonset* pods.
+
+    ```yml
+        securityContext:
+          privileged: true
+    ```
+
+    This is required for the AKV provider pods to successfully startup in ARO.
+
+1. Bind SecurityContextConstraints (SCC) to the Secrets Store CSI Driver and Azure Keyvault Provider service accounts
+
+    ```bash
+    # Replace $target_namespace with the namespace used for helm install
+    # Secrets Store CSI Driver service account
+    oc adm policy add-scc-to-user privileged system:serviceaccount:$target_namespace:secrets-store-csi-driver
```

Comment:
As a followup, it would be good to create a dedicated SCC instead of using `privileged` to ensure we don't grant more than we need.